docs/pages/reference/helm-reference/teleport-plugin-pagerduty.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: teleport-plugin-pagerduty Chart Reference
+description: Values that can be set using the teleport-plugin-pagerduty Helm chart
+---
+
+The `teleport-plugin-pagerduty` Helm chart is used to configure the PagerDuty Teleport plugin, which allows users to receive access requests as pages via PagerDuty.
+
+You can [browse the source on GitHub](https://github.com/gravitational/teleport-plugins/tree/master/charts/access/pagerduty).
+
+This reference details available values for the `teleport-plugin-pagerduty` chart.
+
+(!docs/pages/includes/backup-warning.mdx!)
+
+## `teleport.address`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | Yes |
+
+This parameter contains the host/port combination of the Teleport Auth Service or Proxy Service.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  teleport:
+    address: "teleport.example.com:3025"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set teleport.address="teleport.example.com:3025"
```

Comment:
Is there a reason not to include the binary one would run with this flags in the example?

Reply:
This is the convention we've included in all of our Helm references.